content/docs/04-clusters/06-cluster-management/11.5-kubeconfig.md

```diff
@@ -0,0 +1,99 @@
+
+</InfoBox>
+
+
+The access you have as a user to the kubeconfig file for a cluster depends on the following factors:
+
+<br />
+
+* Is OIDC configured for the cluster? OIDC is configured in the Kubernetes pack YAML file of the cluster profile. Refer to the respective Kubernetes distribution [Pack reference page](/integrations) to learn more about OIDC configuration.
+
+  <br />
+
+  <WarningBox>
+
+    When enabling OIDC, ensure the parameter `oidc-issuer-url` and the `clientConfig` block are properly configured in the Kubernetes Pack YAML. Properly configuring both parameters, you ensure the kubeconfig file is available for all project users. Otherwise, the kubeconfig file will only be available for Cluster Admins or custom roles that have the *delete* permission for the resource key *cluster*.
+
+  </WarningBox>
+
+
+* Is the [Spectro Proxy](/integrations/frp) enabled for the cluster?
+
+
+Use the following tables to help you identify what project role you need to access the kubeconfig file for a cluster. 
+
+
+### Cluster Admin
+
+If a custom Palette resource role is created and used instead of the Palette role *Cluster Admin*. The custom Palette resource role must have the *delete* permissions for the resource key *cluster* to access the kubeconfig file for a cluster.
+
+
+| **Project Role** | **OIDC Configured** | **Spectro Proxy Enabled** | **Access Kubeconfig File** | 
+| --- | --- | --- | --- |
+| Cluster Admin | ✅ | ✅ | ✅ |
+| Cluster Admin | ❌ | ✅ | ✅ |
+| Cluster Admin | ✅ | ❌ | ✅ |
+
+
+
+### Non-Cluster Admin
+
+
+
+*Non-Cluster Admin* refers to lower-level project roles such as the *Cluster Editor* or the *Cluster Viewer*, or custom Palette resource roles that do not have the *delete* permissions for the resource key *cluster*. 
+
+| **Project Role** | **OIDC Configured** | **Spectro Proxy Enabled** | **Access Kubeconfig File** | 
+| --- | --- | --- | --- |
+| Non-Cluster Admin | ✅ | ✅ | ✅ |
+| Non-Cluster Admin | ❌ | ✅ | ❌ |
+| Non-Cluster Admin | ✅ | ❌ | ✅ |
+
+
+# API Access
+
+Palette exposes two API endpoints that you can use to access the kubeconfig file for a cluster. The endpoints are:
+
+<br />
+
+* `GET https://api.spectrocloud.com/v1/spectroclusters/{clusterId}/assets/kubeconfig` - Returns the kubeconfig file for the cluster. The kubeconfig file is returned as a text string. Access to the kubeconfig file is determined by the permissions assigned to you in Palette.  For additional information, refer to the [Kubeconfig Access Permissions](#kubeconfigaccesspermissions) section. 
+
+
+* `GET https://api.spectrocloud.com/v1/spectroclusters/{clusterId}/assets/adminkubeconfig` - Returns the admin kubeconfig file for the cluster. The admin kubeconfig file is returned as a text string. Only users with the Palette project role *Cluster Admin* or with a custom Palette resource role with the resource key *cluster* and the *delete* permission can access the admin kubeconfig file for a cluster.
```

Comment:
Just a suggestion... what if you remove the dash that kind of hangs out there alone after the endpoint, and make the description that starts "Returns ... " its own paragraph indented to align with ``Get https://....``